charts/prometheus-openstack-exporter/values.yaml

```diff
@@ -1,19 +1,40 @@
 ---
 # Choose endpoint type (this will override the value of clouds.yaml)
 endpoint_type: internal
+# Ignored when multicloud.enabled: true
 cloud: default
 
 replicaCount: 1
 
 image:
   repository: quay.io/niedbalski/openstack-exporter-linux-amd64
-  tag: v1.2.0
+  tag: v1.4.0
```

Comment:
But you bumped this here to 1.4.0 -- could you fix to match?

Reply:
Yeah - sorry - my 'local copy for upstream' and my 'local copy I actually run' had diverged in the year since I opened this ;-)
Should be all matching as of the last commit. Thanks for the patience.